.github/workflows/pkgdown.yaml

```diff
@@ -190,31 +192,22 @@ jobs:
         uses: actions/deploy-pages@v4
 
   post-page-artifact:
-    # only comment on PRs
-    if: ${{ github.event_name == 'pull_request' }}
-    needs: combine
     runs-on: ubuntu-latest
+
+    # This job depends on the `build` job
+    needs: combine
+
+    # Required permissions
     permissions:
-        contents: read
-        pull-requests: write
-    env:
-      GH_TOKEN: ${{ github.token }}
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v4
-      - name: Find Comment
-        uses: peter-evans/find-comment@v3
-        id: fc
-        with:
-           issue-number: ${{ github.event.pull_request.number }}
-           comment-author: 'github-actions[bot]'
-           body-includes: Your page is ready to preview
+      contents: read
+      pull-requests: write
 
-      - name: Create or update comment
-        uses: peter-evans/create-or-update-comment@v4
+    steps:
+      # Post the artifact pulling the id from the `readme` step.
+      # The msg will refer to the arfitact as 'README file'.
+      - name: Post the artifact
+        if: ${{ github.event_name == 'pull_request' }}
```

Comment:
why not put the `if` at the `job` level rather than the `step` level, as it was before.